aws-test/tests/aws_config_configuration_recorder/variables.tf

```diff
@@ -7,13 +7,13 @@ variable "resource_name" {
 
 variable "aws_profile" {
   type        = string
-  default     = "integration-tests"
+  default     = "default"
```

Comment:
Did we need to change this? What do other tests use?

Reply:
Ahh, My bad reverted back